app/upgrades/vNEXT.go

```diff
@@ -58,6 +58,10 @@ var zeroBalanceMorseClaimableAccountsJSONBz []byte
 //     The file includes missing accounts from **both** Morse MainNet and TestNet.
 //     It is shared across all networks for simplicity.
 //     There is **zero risk** of unintended token minting (staked or unstaked).
+//
+// - Update the Morse account recovery allowlist:
```

Comment:
Add details regarding how to generate the list.

Reply:
+1